### ISSUE TITLE
Implement support for the Smart Siren

### ISSUE THREAD

**Issue by vlebourl** [jabesq/netatmo-api-python#100]
Hi,
Would it be possible to add support for the new siren (https://www.netatmo.com/en-us/security/cam-indoor/siren)? Like triggering or stopping the siren or the natural sounds to simulate presence?
Cheers
V.

**Comment by cgtobi** [comment]
So far there is no official documentation about an API for the siren. We can "see" it but we can't do anything with it (as far as the official API goes).

**Comment by vlebourl** [comment]
fair enough, thanks for the update! Shall I close the issue for now?

**Comment by cgtobi** [comment]
No, just leave it open. If the API/documentation changes we can pick this up for implementation.

**Comment by jabesq** [comment]
Yes keep it open, so we can contact you if we want to perform some test with the siren once the implementation will start, because I don’t own the siren and I don’t think @cgtobi does too.
